packages/sdk/tests/utils.ts

```diff
@@ -96,3 +97,12 @@ export async function timers(
 		jest.useRealTimers();
 	}
 }
+
+export function generateHash(stringToHash: string): Promise<string> {
+	const buffer = 'string' as unknown as Buffer;
+	const argon2HashConfigOptions = { test: 'test', associatedData: buffer }; // Disallow the HASH_RAW option, see https://github.com/directus/directus/discussions/7670#discussioncomment-1255805
+	// test, if specified, must be passed as a Buffer to argon2.hash, see https://github.com/ranisalt/node-argon2/wiki/Options#test
+	'test' in argon2HashConfigOptions &&
+		(argon2HashConfigOptions.associatedData = Buffer.from(argon2HashConfigOptions.associatedData));
```

Comment:
This syntax could be more readable

```suggestion
	if ('test' in argon2HashConfigOptions)
	  argon2HashConfigOptions.associatedData = Buffer.from(argon2HashConfigOptions.associatedData);
```